Engine/GraphicsCore.cs

```diff
@@ -51,6 +52,9 @@ public GBuffer(int width, int height)
 
         private static bool disposed = false;
 
+        public static event Action<int, int> OnInitialized;
```

Comment:
What for? haven't found any subscriptions

Reply:
Removed it